src/elm/Page/Device.elm

```diff
@@ -140,13 +142,32 @@ update session msg model =
 
         DeviceInfosDone (Err error) ->
             let
-                -- TODO handle error
                 ( message, details ) =
                     AstarteApi.errorToHumanReadable error
+
+                errorMessage =
+                    case error of
+                        AstarteApi.InvalidRequest ->
+                            "Invalid device ID."
+
+                        AstarteApi.Forbidden ->
```

Comment:
I would display "Forbidden." or "Access denied to $DEVICE_ID" or just "Access denied.". @rbino any preference here?